### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by aaemnnosttv**
### Description of the Change

Closes #

### Changelog Entry

> Added - New feature
> Changed - Existing functionality
> Deprecated - Soon-to-be removed feature
> Removed - Feature
> Fixed - Bug fix
> Security - Vulnerability

### Credits

Props @aaemnnosttv 

### Checklist:

- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: wp/headless-wp/includes/classes/Feed.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ public function register() {
 		add_filter( 'the_content_feed', [ $this, 'filter_url' ], 999 );
 		add_filter( 'the_excerpt_rss', [ $this, 'filter_url' ], 999 );
 		add_filter( 'comments_link_feed', [ $this, 'filter_url' ], 999 );
-		add_filter( 'commnet_link', [ $this, 'filter_url' ], 999 );
```
Comment (aaemnnosttv) [inline wp/headless-wp/includes/classes/Feed.php:24]: This one is a bit more significant as the hooked function wouldn't have worked before.

File: packages/core/src/react/blocks/PullQuote.tsx [hunk 2]
```diff
@@ -10,14 +10,14 @@ export interface PullQuoteBlockProps extends IBlockAttributes {
 	borderColor?: string;
 }
 
-export interface IPullQuotekBlock extends IBlock<PullQuoteBlockProps> {}
```
Comment (aaemnnosttv) [inline packages/core/src/react/blocks/PullQuote.tsx:13]: ⚠️

File: packages/core/src/react/blocks/types.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ export interface Typography {
 	supportsCustomFontSize: boolean;
 	supportsFontWeight: boolean;
 	supportsLetterSpacing: boolean;
-	supportsLineHight: boolean;
```
Comment (aaemnnosttv) [inline packages/core/src/react/blocks/types.ts:32]: ⚠️

**Review by aaemnnosttv — COMMENTED** [review]
A few of these changes are functionally-significant. I've flagged them below for review.

File: wp/headless-wp/includes/classes/Feed.php (hunk 1 above)
Comment (nicholasio) [reply wp/headless-wp/includes/classes/Feed.php:24]: Good Catch!

**Review by nicholasio — APPROVED** [review]

**Pull request merged**